middle_end/flambda/inline_and_simplify.ml

```diff
@@ -843,7 +843,7 @@ and simplify_partial_application env r ~lhs_of_application
       }
     in
     let closure_variable =
-      Variable.rename
+      Variable.rename ~debug_info:dbg
```

Comment:
I am not sure that this debuginfo is more relevant than the one on the closure_id.

Reply:
It is now using the debuginfo on closure_info